### DISCUSSION TITLE
Use Cases 用户案例

### DISCUSSION THREAD

**Discussion by jixuan1989** [apache/iotdb#4040] [Show and tell]
Welcome to register by company + homepage (optional) + use case(optional), 
your support is important to us.
We will sync to official website periodically (TBD).

欢迎采用/正在测试IoTDB的公司在此登记，您的支持是我们最大的动力。
请按公司名 + 首页（可选） + 应用案例（可选）的格式登记。
后期我们会定期同步至使用公司列表页面。

**Comment by juice411** [comment]
大唐先一科技，由最初的0.9到目前的0.12，经历了多个版本的测试、反馈、调优、上线、升级，下一步将考虑单机到集群版的升级，大力支持国产软件的推进。

**Comment by kevin-xin** [comment]
金风科技在重点研发项目中使用了iotdb，实时消费kafka中的风机SCADA数据，使用方便接口方便,极值查询，函数计算基本上都是毫秒级返回。

**Comment by zhangweide666** [comment]
北京创智信科，使用IoTDB存储中国铁塔设备监控数据，在试用阶段，IoTDB数据库的插入、存储、查询性能优越，目前正在进行压力和稳定性测试

**Comment by xiamingwu365** [comment]
北京清智物联在分析调相机(发电机)的在线监测数据，测试使用IoTDB，目前在等待分布式版本，用于诊断中心。

**Comment by vesense** [comment]
@jixuan1989 We'd better add these companies to the powered page. This is very convincing for users.
http://iotdb.apache.org/Community/Community-Powered%20By.html#companies-and-organizations

**Comment by lxn1992** [comment]
建龙钢铁，对iotdb进行测试，祝愿乔博等创作者引领世界

**Comment by shareli** [comment]
中泰科技  主要使用iotdb在物联网，智能家居行业。目前在做性能和压力测试。由于没有集群版本，所以还在观望

**Comment by jixuan1989** [comment]
> 中泰科技 主要使用iotdb在物联网，智能家居行业。目前在做性能和压力测试。由于没有集群版本，所以还在观望

可以测试下cluster_new 分支，并非常欢迎对这个分支提issue~

**Comment by chariswang188996** [comment]
太极计算机股份有限公司 企业BG 基础能源SBU 准备 测试性能

**Comment by yuerj** [comment]
用友-云平台-微服务团队，在开发测试环境使用了iotdb，祝愿乔博等创作者引领世界，社区越来越强大

**Comment by chariswang188996** [comment]
太极计算机股份有限公司，目前在做IOTDB的各项性能测试并且在某些场景中进行应用中，在危化应急项目中用来记录危化品车辆的定位信息。祝IOTDB社区越做越强！👍👍

**Comment by fly-chosen** [comment]
开始加入社区，研究

**Comment by xlgo** [comment]
车星际网络科技有限公司      汽车odb盒子，行车记录仪。记录GPS轨迹信息，实现手机查看轨迹，实时位置 等。

**Comment by lizhizhou** [comment]
震坤行 www.zkh360.com 行家设备云 https://iservice.zkh360.com/  测试中

**Comment by LeiRui** [comment]
> @jixuan1989 We'd better add these companies to the powered page. This is very convincing for users.
> http://iotdb.apache.org/Community/Community-Powered%20By.html#companies-and-organizations

Agree. If possible I would suggest listing the cooperating companies on the IoTDB home page. Take [shimo document](https://shimo.im/) as an example:
![image](https://user-images.githubusercontent.com/33376433/119633457-5818b580-be44-11eb-943b-ab4ebda30419.png)

**Comment by Eights-Li** [comment]
长安汽车 IOTDB用于车况信号的存储和查询

**Comment by it-boy-like-girl** [comment]
绍兴安瑞思网络科技，工业互联网项目，对于目前需求，单机完全够用，只希望稳定，查询很高效。nice

**Comment by e1o** [comment]
深圳嘉信公司，使用IoTDB存储武汉桥梁健康监控数据，在试用阶段，IoTDB数据库的插入、存储、查询性能优越，目前在等待Grafana插件，祝愿IotDB越来越好

**Comment by yanghe25413** [comment]
拓维信息系统股份有限公司，使用iotdb为某大型国企采集设备数据，搭建物联网，智能制造相关平台，目前已步入生产环节

**Comment by CloudsYy** [comment]
广东瑞恩科技有限公司，使用iotdb的cnc工厂设备数据采集存储并应用，查询性能各方面还不错，希望后面功能越来越完善！

**Comment by limeng32** [comment]
用友网络科技，多维数据库团队，打算深入研究IoTDB。

**Comment by 38778799** [comment]
四川数安值科技有限公司  将IOTDB 正式应用于数安车联项目中，用于存储车辆状态等数据。感谢IOT DB团队的技术支持，祝愿IOTDB 越来越好，中国基础软件越来越好。

**Comment by liutaohua** [comment]
四维智联([www.autoai.com](http://www.autoai.com)) ，用于车辆数据存储及分析项目，目前已上线分布式版本，2700万+ 测点，每日16亿+条数据入库

**Comment by Superainbower** [comment]
安徽中科美络信息技术有限公司，使用IoTDB存储分析充电桩、新能源车辆数据，祝社区越来越强大

**Comment by shaowtao** [comment]
节点互联，经过多方面调研查询资料，选择目前0.12.4版本，做某电厂生产环境模拟开发，3千个点查询毫秒级返回，流畅不卡顿，目前使用单机版本，争取单机到集群跳跃无故障，支持国产软件。

**Comment by terry8204** [comment]
用于存储 gps轨迹，刚开始测试，感觉不错。后面有结论了再来详细反馈，希望单机版稳定，国产软件崛起。

**Comment by 1186130746** [comment]
北京怡养科技，用于存储智能床垫状态数据，百亿级别的数据。毫秒级别的响应。祝分布式稳定版早日上线

**Comment by wuhuahe2010** [comment]
我们公司主要业务是传统的IT运维，运维指标数据存储目前使用elasticsearch，性能问题一直在折磨我们。由于团队技术能力问题，难以在elastisearch上做代码级优化，目前打算寻找另一种数据库替换。 
咨询一下iotdb在运维领域有没有使用场景的案例供借鉴。

**Comment by slawr** [comment]
Apache IoTDB is used as a vehicle data store in the COVESA [Central Data Service Playground](https://github.com/COVESA/cdsp). 

[COVESA ](https://www.covesa.global/) is an open automotive alliance focused on developing common approaches and technologies for connected vehicles, such as the [VSS](https://covesa.global/project/vehicle-signal-specification/)  data model.

The Central Data Service Playground (CDSP) is a neutral, open playground for data services both within and outside the vehicle in the context of data-centric architectures. It enables investigation into the internals of these services and how they can be combined. Furthermore, the playground provides a means to publish and collaborate on such work in the open.

In addition, W3C and COVESA co-developed the automotive [VISS ](https://covesa.global/project/vehicle-information-service-specification/) API specification for accessing VSS data and the [VISSR ](https://covesa.global/project/vehicle-information-service-specification-reference/)project which maintains a reference implementation. As part of the Playground project support for using IoTDB as a data back-end for the reference implementation has been upstreamed
